src/wp-includes/script-loader.php

```diff
@@ -2267,38 +2267,51 @@ function wp_enqueue_global_styles() {
 		return;
 	}
 
+	$separate_assets = wp_should_load_separate_core_block_assets();
+
+	// Global styles should be displayed in the header when loading combined.
```

Comment:
Can we add a link to the ticket https://core.trac.wordpress.org/ticket/53494 so it's obvious why this was added? The comment, as it is, may mislead future selves because for block-based themes (still not in core) we still want to enqueue the styles in the header. It's only the classic theme with separate core assets enabled.

Reply:
Should we also add this clarification to `wp_should_load_separate_core_block_assets()`? How that function behaves is now well documented, but not really when it should be used.